Integrations/DigitalDefense/DigitalDefense.yml

```diff
@@ -0,0 +1,160 @@
+  version: -1
+name: Digital Defense Inc. FrontlineVM
+display: Digital Defense Inc. FrontlineVM
+category: Vulnerability Management
+description: Digital Defense Inc.'s Frontline Vulnerability Management solution comprehensively
+  identifies and evaluates the security and business risk postures of network devices
+  and applications deployed as premise, cloud, or hybrid network-based implementations.
+  Now residing entirely in Amazon Web Services (AWS), Frontline VM easily addresses
+  the security compliance requirements of organizations around the globe.
+configuration:
+- display: API Token to access Frontline.Cloud
+  name: apiToken
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Min vulnerability severity for fetching incidents.
+  name: incidentSeverity
+  defaultvalue: critical
+  type: 15
+  required: false
+  options:
+  - critical
+  - high
+  - medium
+  - low
+  - trivial
+  - info
+- display: Rate at which to check vulnerability events when fetching incidents.
+  name: incidentFrequency
+  defaultvalue: 3 hr
+  type: 15
+  required: true
+  options:
+  - 5 min
+  - 15 min
+  - 30 min
+  - 1 hr
+  - 3 hr
+  - 8 hr
+  - 12 hr
+  - 24 hr
+script:
+  script: ''
+  type: python
+  commands:
+  - name: frontline-get-assets
+    arguments:
+    - name: ip_address
+      description: Get assets for given IP Address
+    - name: label_name
+      description: Get assets with given label name (put name in quotes if spaces
+        are required).
+    - name: scanned_within_last
+      auto: PREDEFINED
+      predefined:
+      - "1"
+      - "7"
+      - "30"
+      - "60"
+      - "90"
+      - "180"
+      description: Get hosts scanned within the given range (in days)
+    - name: hostname
+      description: Get asset for given hostname
+    outputs:
+    - contextPath: FrontlineVM.Hosts
+      description: Returned host data pulled from Frontline.Cloud
+      type: unknown
+    - contextPath: FrontlineVM.IPList
+      description: IP address list of found hosts.
+      type: unknown
+    - contextPath: FrontlineVM.HostId
+      description: ID number of Host pulled
+    description: Pull asset information from FrontlineVM
+  - name: frontline-get-vulns
+    arguments:
+    - name: min_severity
+      auto: PREDEFINED
+      predefined:
+      - critical
+      - high
+      - medium
+      - low
+      - trivial
+      - info
+      description: Select a minimum severity level of vulnerabilities to pull from
+        Frontline. E.g. setting the min_severity=medium will pull all vulnerabilities
+        with severity levels of medium, high, and critical.
+    - name: severity
+      auto: PREDEFINED
+      predefined:
+      - critical
+      - high
+      - medium
+      - low
+      - trivial
+      - info
```

Comment:
It will be easier (performance and automation wise) keeping the option to pick the minimum severity level.
Reason being is based on our API, if a user selects multiple severities from a multi-choice selection we would have to iterate though the list, make a GET request for every option they choose with the query parameter `?_0_eq_vuln_severity_ddi=<severity>`, and combine the results of each query into the results.

Whereas if the user chooses a minimum severity, we make only one GET request with the query parameter `lte_vuln_severity_ddi=<min_severity>` and return those results as is. This request will pull vulnerabilities with multiple severities without having to iterate through each severity chosen from the multi-choice list and will also only need to perform one GET request in the process.